### PR TITLE
improve changelog render

### PR DESCRIPTION
**Pull request by Alexandre-Herve**
## 🍗 Description
_What does this PR do? Anything folks should know?_

- improve wording
- add support for enums
- misc fixes

## 📚 References
_Links to relevant docs (Notion, Twist, GH issues, etc.), if applicable._

## 👹 QA
_How can other humans verify that this PR is correct?_

### CODE REVIEW

File: projects/optic/src/__tests__/integration/__snapshots__/history.test.ts.snap [hunk 1]
```diff
@@ -4,93 +4,93 @@ exports[`optic history writes changelog 1`] = `
...
 ### Sun Jan 01 2023
 - \`PUT\` \`/user/{username}\`:
-  - removed \`requestBody\` \`phone\` property
-  - added \`requestBody\` \`bio\` property
-  - changed \`requestBody\` \`userStatus\`.\`type\`
-  - removed \`requestBody\` \`userStatus\`.\`format\`
-  - added \`requestBody\` \`userStatus\`.\`enum\`
-  - removed \`requestBody\` \`lastName\` as required
-  - added \`requestBody\` \`username\` as required
+  - \`requestBody\`: removed support for \`phone\` property
```
Comment (Alexandre-Herve) [inline projects/optic/src/__tests__/integration/__snapshots__/history.test.ts.snap:7]: @acunniffe what do you think about those changes

File: projects/optic/src/__tests__/integration/__snapshots__/history.test.ts.snap [hunk 2]
```diff
@@ -4,93 +4,93 @@ exports[`optic history writes changelog 1`] = `
...
+  - \`requestBody\`: added support for \`bio\` property
+  - \`requestBody\`: changed \`userStatus\`.\`type\`
+  - \`requestBody\`: removed \`userStatus\`.\`format\`
+  - \`requestBody\`: added \`userStatus\`.\`enum\`
+  - \`requestBody\`: \`lastName\` is not required anymore
```
Comment (acunniffe) [inline projects/optic/src/__tests__/integration/__snapshots__/history.test.ts.snap:12]: `is no longer required` is a bit more natural

File: projects/optic/src/__tests__/integration/__snapshots__/history.test.ts.snap [hunk 3]
```diff
@@ -4,93 +4,93 @@ exports[`optic history writes changelog 1`] = `
...
 - \`POST\` \`/store/order\`:
-  - added \`200\` response \`summary\` property
-  - removed \`200\` response \`status\`.\`enum\`.\`1\`
-  - added \`200\` response \`status\`.\`enum\`.\`2\`
-  - added \`requestBody\` \`summary\` property
-  - removed \`requestBody\` \`status\`.\`enum\`.\`1\`
-  - added \`requestBody\` \`status\`.\`enum\`.\`2\`
+  - \`200\` response: added support for \`summary\` property
+  - \`200\` response: removed support for value \`approved\` on enum \`status\`
+  - \`200\` response: added support for new value \`canceled\` on enum \`status\`
```
Comment (acunniffe) [inline projects/optic/src/__tests__/integration/__snapshots__/history.test.ts.snap:62]: hmmm I feel like it's `added enum option 'canceled' on 'status'` and `removed enum option approved on 'status'`

File: projects/optic/src/__tests__/integration/__snapshots__/history.test.ts.snap [hunk 4]
```diff
@@ -4,93 +4,93 @@ exports[`optic history writes changelog 1`] = `
...
-  - removed \`200\` response \`stringOrNumberOrObject\`.\`oneOf\`.\`2\`
+  - \`200\` response: changed \`toTypeArrays\`.\`type\`
+  - \`200\` response: changed \`fromTypeArrays\`.\`type\`
+  - \`200\` response: \`requiredKeys\`.\`requiredToOptional\` is not required anymore
+  - \`200\` response: \`requiredKeys\`.\`deletedRequiredKey\` is not required anymore
+  - \`200\` response: \`requiredKeys\`.\`optionalToRequired\` is now required
+  - \`200\` response: \`requiredKeys\`.\`addedRequiredKey\` is now required
+  - \`200\` response: removed support for \`requiredKeys\`.\`deletedRequiredKey\` property
+  - \`200\` response: added support for \`requiredKeys\`.\`addedRequiredKey\` property
+  - \`200\` response: added \`expandableObject\`.\`anyOf\`.\`1\`
```
Comment (acunniffe) [inline projects/optic/src/__tests__/integration/__snapshots__/history.test.ts.snap:91]: I'm not sure what this is trying to say.

File: projects/optic/src/__tests__/integration/__snapshots__/history.test.ts.snap (hunk 3 above)
Comment (Alexandre-Herve) [reply projects/optic/src/__tests__/integration/__snapshots__/history.test.ts.snap:62]: I can change but just to let you know I took the Stripe wording there

**Review by niclim — APPROVED** [review]

**Pull request merged**